examples/learn/calls/basic_call/mistral/official_sdk_call.py

```diff
@@ -1,6 +1,7 @@
-from mistralai.client import MistralClient
+from mistralai import Mistral
+import os
 
-client = MistralClient()
+client = Mistral(api_key=os.environ.get("MISTRAL_API_KEY"))
```

Comment:
we should let the client load the api key like we do for other providers, so just client = Mistral() should suffice here, no?